tox.ini

```diff
@@ -185,3 +174,16 @@ commands =
         --cache-dir="{toxworkdir}/mypy_cache"  \
         {tty:--pretty:}                        \
         {posargs:src}
+
```

Comment:
Can you please add a comment explaining the purpose / goal of this tox env?

I am not sure why we need this.

It looks like it creates an sdist based on PEP517 and together with the wheel created tox it will validate the files.

Will a wheel be created if we just run release, or it depends on a previous build?

Reply:
> Can you please add a comment explaining the purpose / goal of this tox env?
> 
> I am not sure why we need this.

it copies the wheel from the tox dist directory to the "./dist" directory for the upload

> It looks like it creates an sdist based on PEP517 and together with the wheel created tox it will validate the files.
> 
> Will a wheel be created if we just run release, or it depends on a previous build?

yes a wheel will be created and checked if you run `tox -e release`